File: Event_graduation/Event_graduation.Aplication/DTOS/Response/EventResponseDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Event_graduation.Aplication.DTOS.Response
{
    public class EventResponseDto
    {

        public int Id { get; set; }
        public string InstitutionName { get; set; }
        public string InstitutionAddress { get; set; }
        public int NumberOfStudents { get; set; }
        public DateTime StartTime { get; set; } 
        public decimal ServicePrice { get; set; }
        public string Email { get; set; }
        public DateTime? CreationDate { get; set; }

        public DateTime? UpdateDate { get; set; }
    }
}

File: event/src/app/event/dtos/event-response-dto.ts
export class EventResponseDto {
  public id: number;
  public institutionName: string;
  public institutionAddress: string;
  public numberOfStudents: number;
  public startTime: Date;
  public servicePrice: number;
  public email: string;
  public creationDate?: Date;
  public updateDate?: Date;

  constructor(
    id: number,
    institutionName: string,
    institutionAddress: string,
    numberOfStudents: number,
    startTime: Date,
    servicePrice: number,
    email: string,
    creationDate?: Date,
    updateDate?: Date
  ) {
    this.id = id;
    this.institutionName = institutionName;
    this.institutionAddress = institutionAddress;
    this.numberOfStudents = numberOfStudents;
    this.startTime = startTime;
    this.servicePrice = servicePrice;
    this.email = email;
    this.creationDate = creationDate;
    this.updateDate = updateDate;
  }
}
